napalm_ios/ios.py

```diff
@@ -142,23 +145,66 @@ def is_alive(self):
             'is_alive': self.device.remote_conn.transport.is_active()
         }
 
+    @staticmethod
+    def _create_tmp_file(config):
+        """Write temp file and for use with inline config and SCP."""
+        tmp_dir = os.getcwd()
```

Comment:
@mirceaulinic Okay, that seems reasonable to me:

```
When set to a value other than None, this variable defines the default value for the dir argument to all the functions defined in this module.

If tempdir is unset or None at any call to any of the above functions, Python searches a standard list of directories and sets tempdir to the first one which the calling user can create files in. The list is:

The directory named by the TMPDIR environment variable.
The directory named by the TEMP environment variable.
The directory named by the TMP environment variable.
A platform-specific location:
On RiscOS, the directory named by the Wimp$ScrapDir environment variable.
On Windows, the directories C:\TEMP, C:\TMP, \TEMP, and \TMP, in that order.
On all other platforms, the directories /tmp, /var/tmp, and /usr/tmp, in that order.
As a last resort, the current working directory.
```

As it would default to /tmp (usually), but you could overwrite it by environment variables.